mod/wall_upload.php

```diff
@@ -119,6 +119,7 @@ function wall_upload_post(&$a, $desktopmode = true) {
 	$maximagesize = get_config('system','maximagesize');
 
 	if(($maximagesize) && ($filesize > $maximagesize)) {
+		logger("Image exceeds size limit of ".$maximagesize);
 		echo  sprintf( t('Image exceeds size limit of %s'), formatBytes($maximagesize)) . EOL;
```

Comment:
Without having read the rest of the file, but should not be the echo a call for info() or a logger?